### ISSUE TITLE
How can I make it work with 64 bits MetaTrader5?

### ISSUE THREAD

**Issue by luismasuelli** [tickelton/docker-metatrader#3]
I downloaded MetaTrader5 from Roboforex, and it is a 64 bits software. Names like "terminal64.exe" and "metatester64.exe" are among the filenames instead of their 32 bits counterparts. When I try running them I receive these errors:

0024:err:module:process_init L"Z:\\path\to\terminal64.exe" 64-bit application not supported in 32-bit prefix.

How can I tweak the Dockerfile to use 64 bits instead of 32 bits MetaTrader?

**Comment by tickelton** [comment]
At the time I created the Dockerfile the 64 bit version MetaTrader5 would
not run in Wine because of compatibility issues.

If those have been solved in newer Wine version, it might be enough to
remove the following line from the Dockerfile:
ENV WINEARCH win32

You might also have to use a newer version of Wine and the base Image for
that, tough. So you would have to replace "xenial" in the following two
lines with "groovy":

FROM ubuntu:xenial

RUN echo "deb http://dl.winehq.org/wine-builds/ubuntu/ xenial main" >>
/etc/apt/sources.list && \

I haven't tried any of this myself because I still use an older 32 bit
version of MetaTrader, but I'll give it a try later this week.
